pkg/readiness/ready_tracker.go

```diff
@@ -46,7 +45,9 @@ import (
 
 var log = logf.Log.WithName("readiness-tracker")
 
-var crashOnFailureFetchingExpectations = flag.Bool("crash-on-failure-fetching-expectations", false, "Unless set (defaults to false), gatekeeper will ignore errors when gathering expectations. This prevents bootstrapping errors from crashing Gatekeeper at the cost of increasing the risk Gatekeeper will under-enforce policy. Enabling this will help prevent under-enforcement at the risk of crashing during startup. Note that enabling this flag currently does not achieve the aforementioned effect since fetching expectations will retry until success.")
+// Commenting out the flag and replacing with a false boolean constant because the value of the flag is currently moot without a retry limit
+// var crashOnFailureFetchingExpectations = flag.Bool("crash-on-failure-fetching-expectations", false, "Unless set (defaults to false), gatekeeper will ignore errors when gathering expectations. This prevents bootstrapping errors from crashing Gatekeeper at the cost of increasing the risk Gatekeeper will under-enforce policy. Enabling this will help prevent under-enforcement at the risk of crashing during startup. Note that enabling this flag currently does not achieve the aforementioned effect since fetching expectations will retry until success.")
+const crashOnFailureFetchingExpectations = false
```

Comment:
Can't do &false. I could do something like 
var falseValue = false
var crashOnFailureFetchingExpectations = &falseValue, but seemed unnecessary

Reply:
Add a TODO as a comment and open issue to track so we remember to add the flag back would be great!